.github/workflows/base-release.yaml

```diff
@@ -150,7 +150,7 @@ jobs:
           distribution: goreleaser-pro
           version: latest
           workdir: distributions/${{ inputs.distribution }}
-          args: continue --merge --timeout 2h
+          args: continue --merge --timeout 2h --release-header-tmpl=../.github/release-template.md
```

Comment:
The working dir is a folder in `distributions`, so I believe you need another `../`

Reply:
@TylerHelmuth I tried that; it only works if we are pointing directly to one folder within the dir, like we did in the second test. Since it's `distributions/${{ inputs.distribution }}`, just `../` works.
Here's a link to where I tested: https://github.com/Enzujp/sample-releases/actions/runs/8716988185/job/23911398530

Reply:
@Enzujp the reason it worked in your test is because `${{ inputs.distribution }}` didn't exist so your working dir was `distributions/`

![image](https://github.com/open-telemetry/opentelemetry-collector-releases/assets/12352919/39e1eefa-8db5-43b1-bff6-2f95c848dd33)

Reply:
oh, okay. Thank you. I'll fix it now.

Reply:
I just tested this in my repo, we need to move the flag to the actual `release` command instead of the `continue` command.

Reply:
Example: https://github.com/TylerHelmuth/opentelemetry-collector-releases/actions/runs/8726155596

Reply:
I just took a look, I'll do that now.

Reply:
@TylerHelmuth I've done that .